config/manager/kustomization.yaml

```diff
@@ -6,6 +6,8 @@ resources:
 
 generatorOptions:
   disableNameSuffixHash: true
+  labels:
```

Comment:
I believe we've chatted about why this label is added: because we're starting the watcher in the operator even when the helm chart has the revocation logic explicitly disabled. I think that's a separate discussion and we can iterate on that approach later on. But, adding this one label to the manager and none of the other kustomize resources seems a bit odd. For reference, helm adds it to all resources not just the deployment.
nit: you probably don't need to go and add the label to every resource in the config directory, but if there is an easy one-line approach to adding a label to all resources in Kustomize I'd like that instead :)

Reply:
Thank you! I've switch to this one-line approach https://github.com/hashicorp/vault-secrets-operator/blob/371ef5d9042e7739dfca1356e7be642fdda0ba70/config/default/kustomization.yaml#L15-L16